src/dcat-us/index.test.ts

```diff
@@ -7,12 +7,14 @@ import * as mockSiteModel from '../test-helpers/mock-site-model.json';
 import { DcatDatasetTemplate } from './dataset-formatter';
 import { IItem } from '@esri/arcgis-rest-types';
 
+const hostname = 'css-monster-qa-pre-hub.hubqa.arcgis.com';
+
 async function generateDcatFeed(
-  siteItem: IItem,
+  _siteItem: IItem,
```

Comment:
Go ahead and remove this argument since you're using the constant hostname instead

Reply:
done